crates/build/src/lib.rs

```diff
@@ -204,8 +207,12 @@ impl BuildResult {
         );
         if let Some(metadata_result) = self.metadata_result.as_ref() {
             let bundle = format!(
-                "  - {} (code + metadata)\n",
-                util::base_name(&metadata_result.dest_bundle).bold()
+                "  - {} ({})\n",
+                util::base_name(&metadata_result.dest_bundle).bold(),
+                match metadata_result.spec {
+                    MetadataSpec::Ink => "code + metadata",
+                    MetadataSpec::Solidity => "Solidity compatible contract metadata",
```

Comment:
```suggestion
                    MetadataSpec::Solidity => "Solidity compatible metadata",
```

For a consistent naming throughout.